clients/java/src/main/java/io/camunda/client/api/search/filter/builder/ProcessInstanceStateProperty.java

```diff
@@ -15,12 +15,12 @@
  */
 package io.camunda.client.api.search.filter.builder;
 
-import io.camunda.client.protocol.rest.ProcessInstanceStateEnum;
+import io.camunda.client.api.search.response.ProcessInstanceState;
 import io.camunda.client.protocol.rest.ProcessInstanceStateFilterProperty;
 
 public interface ProcessInstanceStateProperty
     extends LikeProperty<
-        ProcessInstanceStateEnum,
+        ProcessInstanceState,
```

Comment:
💭 I noticed that this builder was relying on the API enum instead of our own. Fixed it.

Reply:
Will also be handled with https://github.com/camunda/camunda/pull/29792. @mihail-ca, just for awareness. Whoever merges first, probably "wins" here. Might lead to a merge conflict for the other one.